packages/ember-metal/tests/run_loop/later_test.js

```diff
@@ -79,15 +92,7 @@ asyncTest('should always invoke within a separate runloop', function() {
       secondRunLoop = run.currentRunLoop;
     }, 10, 1);
 
-    // Synchronous "sleep". This simulates work being done
-    // after run.later was called but before the run loop
-    // has flushed. In previous versions, this would have
-    // caused the run.later callback to have run from
-    // within the run loop flush, since by the time the
-    // run loop has to flush, it would have considered
-    // the timer already expired.
```

Comment:
Was it okay to move this comment when I created the `pauseUntil` function? I left the other comment alone because I don't know much about this code so perhaps somebody can advise?